### PR TITLE
(minikube) Add Minikube to Core team packages

### PR DESCRIPTION
**Pull request by abejenaru**
Migrate Minikube to Core team packages.
Adapted to make use of AU for automatic update.

## How Has this Been Tested?
Tested both locally and in Test environment VM.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Migrated package (a package have been migrated from another repository)

## Checklist:
- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this mean usually the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this mean usually the notes in the description of a package).
- [x] All files is up to date with the latest [Contributing Guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md)
- [x] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

## Original Location
- [Original Repository](https://github.com/abejenaru/chocolatey-packages/tree/master/automatic/minikube)
- [ ] *Include the link to the opened PR that removes the package from the original location*
- [x] The [migration guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Package-migration-process) have been followed

### CODE REVIEW

File: automatic/minikube/update.ps1 [hunk 1]
```diff
@@ -0,0 +1,46 @@
+[CmdletBinding()]
+param([switch] $Force)
+
+Import-Module AU
+
+$domain   = 'https://github.com'
+$releases = "$domain/kubernetes/minikube/releases/latest"
+
+function global:au_BeforeUpdate {
+  Get-RemoteFiles -Purge -NoSuffix
```
Comment (AdmiringWorm) [inline automatic/minikube/update.ps1]: add the following parameter to this call:
`-FileNameBase minikube`

Then you don't need to rename the executable below.

File: automatic/minikube/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+### Minikube Features
+Minikube supports Kubernetes features such as:
+* DNS
+* NodePorts
+* ConfigMaps and Secrets
+* Dashboards
+* Container Runtime: Docker, rkt and CRI-O
+* Enabling CNI (Container Network Interface)
+* Ingress
```
Comment (AdmiringWorm) [inline automatic/minikube/README.md]: Please add a notes section that mentions that the software is only for 64bit systems.

File: automatic/minikube/legal/VERIFICATION.txt [hunk 3]
```diff
@@ -0,0 +1,22 @@
+VERIFICATION
+
+Verification is intended to assist the Chocolatey moderators and community
+in verifying that this package's contents are trustworthy.
+
+The embedded software have been downloaded from the listed download
+location on <https://github.com/kubernetes/minikube/releases/tag/v0.24.1>
```
Comment (AdmiringWorm) [inline automatic/minikube/legal/VERIFICATION.txt]: the download location doesn't match the actual location used for the executable.
Either this location needs to be changed, or the file will need to be downloaded from github.

Comment (abejenaru) [reply automatic/minikube/legal/VERIFICATION.txt]: Github binary doesn't end in ".exe" and messes Get-RemoteFiles; the alternative link which I use is mentioned under __Distribution__ (*Binaries are available through GitHub or on Google Cloud Storage. The direct GCS links are*)

Comment (AdmiringWorm) [reply automatic/minikube/legal/VERIFICATION.txt]: Then the VERIFICATION.txt file needs to reflect which files on the github repo is equal to the downloaded binaries.

**Pull request merged**